docs/getting-started/backends-and-brokers/sqs.rst

```diff
@@ -192,6 +192,27 @@ The above policy:
 +-----------------------------------------+--------------------------------------------+
 
 
+SQS - STS token authentication
```

Comment:
Please provide a link to AWS's documentation regarding the matter.

Reply:
added